WithSecure/action_update_incident_status.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Update status on Incident",
+  "description": "Update status on Incident.",
+  "uuid": "0410768f-8d21-4aac-94d7-9ef62a8d7cfa",
+  "docker_parameters": "update_incident_status",
+  "arguments": {
+    "title": "Incident",
+    "type": "object",
+    "properties": {
+      "target": {
+        "title": "Target",
```

Comment:
For the 3 actions, I think that `incident_id` would be more meaningful for the customers

Reply:
Yes, I change that

Reply:
Done